portfolio/src/main/webapp/index.html

```diff
@@ -8,28 +8,30 @@
   </head>
   <body>
     <div id="content">
-      <a href="index.html">Home</a>
-      <p><a href="about-me.html">About Me</a> -- Click here to learn more about me</p>
-      <p><a href="baking.html">Macarons and Things</a> -- A food gallery of my best baking accomplishments</p>
+      <ul>
+        <li><a href="index.html" class="active">Home</a></li>
+        <li><a href="about-me.html">About Me</a></li>
+        <li><a href="baking.html">Food</a></li>
+      </ul>
       <h1>Megan Shi</h1>
-      <img src="images/me.JPG" height=250 alt="Photo of Me">
+      <img src="images/me.JPG" height=250 alt="Photo of Me" class="center">
       <p>Hi, I'm Megan! I'm a rising junior at the University of Washington studying CS. 
         I like food (making AND eating) and love trying new recipes. In my free time, 
         I like playing tennis with my mom and video games with my friends. I'm looking forward to learning
         about software engineering and building projects. I also hope to build a PC this 
         summer so if you have any guidance let me know :^)</p>
       <p>
       <p>I also have two cats named Buzz and Ozzy!</p>
-      <img src="/images/cats.JPG" id="cat-image" height=250 alt="My cats Buzz and Ozzy">
-      <div>
-        <button onclick="changeCatPhoto()">click me to get a random photo of my cats!</button>
+      <img src="/images/cats.JPG" id="cat-image" height=250 alt="My cats Buzz and Ozzy" class="center">
+      <div class="centered-button">
```

Comment:
Is this div needed? The button class looks like it already has CSS for text-align: center.

Reply:
I tried it out in my code and I think having the <div> with the corresponding css text-align: center is necessary, but the text-align: center in the button class isn't necessary. Is it okay to just get rid of the text-align: center in the button class instead?

Reply:
Seems strange that the text isn't centering on the button unless you put it in the div container...
What about: `<button class="centered-button">` ? The button element CSS should still apply

Reply:
To clarify, I'm using the div container and its corresponding text-align: center to make the button centered on the screen, and I believe the margin-left/right: auto in the button component centers the button's text within the button (but centering the text within the button wasn't really my focus here). Is there a better way than using the div to center the button on the screen?

Reply:
Thanks for clarifying! There are other ways to do it (https://medium.com/better-programming/how-to-center-things-with-style-in-css-dc87b7542689) but this seems like the simplest. I'm not sure style-wise if one is preferred over the other but we can keep it as this